BuildResidentialHPXML/tests/base-bldgtype-single-family-attached-2stories.osw

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
Adds a test file that differs between here and the base branch. All other test files show identical results.